cli/src/main/scala/org/apache/celeborn/cli/master/MasterSubcommandImpl.scala

```diff
@@ -0,0 +1,187 @@
+ * limitations under the License.
+ */
+
+package org.apache.celeborn.cli.master
+
+import java.util
+
+import scala.collection.JavaConverters._
+
+import picocli.CommandLine.{Command, ParameterException}
+
+import org.apache.celeborn.cli.config.{CliConfig, CliConfigManager}
+import org.apache.celeborn.rest.v1.model._
+import org.apache.celeborn.rest.v1.model.SendWorkerEventRequest.EventTypeEnum
+
+@Command(name = "master", mixinStandardHelpOptions = true)
+class MasterSubcommandImpl extends Runnable with MasterSubcommand {
+  override def run(): Unit = {
+    if (masterOptions.showMastersInfo) log(runShowMastersInfo)
+    if (masterOptions.showClusterApps) log(runShowClusterApps)
+    if (masterOptions.showClusterShuffles) log(runShowClusterShuffles)
+    if (masterOptions.showTopDiskUsedApps) log(runShowTopDiskUsedApps)
+    if (masterOptions.excludeWorkers) log(runExcludeWorkers)
+    if (masterOptions.removeExcludedWorkers) log(runRemoveExcludedWorkers)
+    if (masterOptions.sendWorkerEvent != null && masterOptions.sendWorkerEvent.nonEmpty)
+      log(runSendWorkerEvent)
+    if (masterOptions.showWorkerEventInfo) log(runShowWorkerEventInfo)
+    if (masterOptions.showLostWorkers) log(runShowLostWorkers)
+    if (masterOptions.showExcludedWorkers) log(runShowExcludedWorkers)
+    if (masterOptions.showShutdownWorkers) log(runShowShutdownWorkers)
+    if (masterOptions.showLifecycleManagers) log(runShowLifecycleManagers)
+    if (masterOptions.showWorkers) log(runShowWorkers)
+    if (masterOptions.showConf) log(runShowConf)
+    if (masterOptions.showDynamicConf) log(runShowDynamicConf)
+    if (masterOptions.showThreadDump) log(runShowThreadDump)
+    if (masterOptions.addClusterAlias != null && masterOptions.addClusterAlias.nonEmpty)
+      runAddClusterAlias
+    if (masterOptions.removeClusterAlias != null && masterOptions.removeClusterAlias.nonEmpty)
+      runRemoveClusterAlias
+  }
+
+  private[master] def runShowMastersInfo: MasterInfoResponse = masterApi.getMasterGroupInfo
+
+  private[master] def runShowClusterApps: ApplicationsHeartbeatResponse =
+    applicationApi.getApplications
+
+  private[master] def runShowClusterShuffles: ShufflesResponse = shuffleApi.getShuffles
+
+  private[master] def runShowTopDiskUsedApps: AppDiskUsageSnapshotsResponse =
+    applicationApi.getApplicationsDiskUsageSnapshots
+
+  private[master] def runExcludeWorkers: HandleResponse = {
+    val workerIds = getWorkerIds
+    val excludeWorkerRequest = new ExcludeWorkerRequest().add(workerIds)
+    logInfo(s"Sending exclude worker requests to workers: $workerIds")
+    workerApi.excludeWorker(excludeWorkerRequest)
+  }
+
+  private[master] def runRemoveExcludedWorkers: HandleResponse = {
+    val workerIds = getWorkerIds
+    val removeExcludeWorkerRequest = new ExcludeWorkerRequest().remove(workerIds)
+    logInfo(s"Sending remove exclude worker requests to workers: $workerIds")
+    workerApi.excludeWorker(removeExcludeWorkerRequest)
```

Comment:
for this kind of POST requests, seems we have to call the celeborn master leader

Reply:
Agreed. I feel this should be validation on the server side, not client side. We should send an error message (not 200), if the leader is not being used.

Reply:
I can create another PR for that after this PR @turboFei

Reply:
ok, Sounds good